src/targets/unified.ini

```diff
@@ -66,10 +66,6 @@ extends = env:Unified_ESP8285_900_TX_via_UART
 extends = env_common_esp32, radio_LR1121
 build_flags =
 	${env_common_esp32.build_flags}
-	${radio_LR1121.build_flags}
-	${common_env_data.build_flags_tx}
-	-include target/Unified_ESP32_TX.h
-	-D TARGET_AXIS_THOR_2400_TX=1
```

Comment:
You were just meant to remove the AXIS line

Reply:
I was about to say it doesn't work anymore, 3rd times the charm.